theme/models.py

```diff
@@ -8,6 +8,7 @@
 from django.utils.translation import ugettext_lazy as _
 from django.utils.html import strip_tags
 from django.core.exceptions import ValidationError
+from django.contrib.postgres.fields import HStoreField
```

Comment:
Are we ever going to be indexing these identifiers? @pkdash Might we want to use a JsonField instead here? or, what was the rationale for using HStore over a JsonField?

Reply:
@aphelionz Good question related to indexing. I am not sure. I don't see the need for searching anything based on identifiers. I would love to use JsonField over HStore. However, JsonField is available in Django 1.11 or later version. Once we start using a Django version that supports JsonField, we may be able to use a single JsonField to store all metadata instead of  so many tables - one table for a specific metadata element.

Reply:
@pkdash ahhhh right. My apologies. Yes, I would _love_ that as well. Approved.